### PR TITLE
feat: support class

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

schedule can extend from Subscription which is context class

Ref https://github.com/eggjs/egg/issues/1468

### CODE REVIEW

File: test/fixtures/subscription/app/schedule/interval.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const Subscription = require('egg').BaseContextClass;
```
Comment (atian25) [inline test/fixtures/subscription/app/schedule/interval.js]: egg 那个合掉？

**Comment by popomore** [comment]
改了

**Comment by atian25** [comment]
@popomore README 和 egg docs 也要改下

**Comment by popomore** [comment]
等下再提个 PR

**Pull request merged**